### DISCUSSION TITLE
[BUDI-7349] SignatureDoesNotMatch and 304 Forbidden Errors on Docker Compose Install

### DISCUSSION THREAD

**Discussion by bocajspear1** [Budibase/budibase#11414] [Help]
**Checklist**

* [X] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**

* Self
  * Method: docker compose
  * Budibase Version: 2.8.10

**Describe the bug**
Getting `SignatureDoesNotMatch` with 403 Forbidden error when trying to access `/files/signed/prod-budi-app-assets/app_c337ef7afb604c8dbd9fdae07c5332d1/budibase-client.js`:

```
<Error>
<Code>SignatureDoesNotMatch</Code>
<Message>
The request signature we calculated does not match the signature you provided. Check your key and signing method.
</Message>
<Key>
signed/prod-budi-app-assets/app_c337ef7afb604c8dbd9fdae07c5332d1/budibase-client.js
</Key>
<BucketName>files</BucketName>
<Resource>
/files/signed/prod-budi-app-assets/app_c337ef7afb604c8dbd9fdae07c5332d1/budibase-client.js
</Resource>
<RequestId>1771369741038D7D</RequestId>
<HostId>4931b22b-1cd3-43b4-bc1c-f09c3ccf4ea8</HostId>
</Error>
```

The websocket connection is also failing with a 403 Forbidden error, leading to error pop ups occasionally.

**To Reproduce**

* Created `docker-compose.yml`
* Created `.env` file and updated `BB_ADMIN_USER_EMAIL` and `BB_ADMIN_USER_PASSWORD`.
* Accessed UI at `localhost:10000`
* Clicked though tutorial
* Attempted to use Design tab

**Expected behavior**
Should not be getting 403 errors and be able to use builder.

**Screenshots**

![](https://uploads.linear.app/7ecfa297-cc77-4fdf-9117-fa5afdb8f908/f2c2af0a-478b-41b9-83b1-e37d3fec40b2/c3389fea-aaa9-4de2-8f12-62664bc05564?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzdlY2ZhMjk3LWNjNzctNGZkZi05MTE3LWZhNWFmZGI4ZjkwOC9mMmMyYWYwYS00NzhiLTQxYjktODNiMS1lMzdkM2ZlYzQwYjIvYzMzODlmZWEtYWFhOS00ZGUyLThmMTItNjI2NjRiYzA1NTY0IiwiaWF0IjoxNjkwODk2OTQwLCJleHAiOjE2OTA5ODMzNDB9.DfV9Vja6jEtOJ2hSbd9G7jFKGkMrulp1YPjx3JnNzGE)

**Desktop:**

* OS: Ubuntu Linux
* Browser: Firefox
* Version: 115.0.2

**Additional context**

* Installing in VM that commonly has time out of sync due to me saving the VM state to disk. However, made sure time was synced and even rebooted the VM to ensure that was not the issue.
* I've tried doing a clean execution (running `docker compose down -v` and then `docker compose up`) multiple times with no success.
* Attempted with `127.0.0.1` and `localhost`
* Cleared cache and cookies as well in Firefox

<sub>From [SyncLinear.com](https://synclinear.com) | [BUDI-7349](https://linear.app/budibase/issue/BUDI-7349/signaturedoesnotmatch-and-304-forbidden-errors-on-docker-compose)</sub>

**Comment by melohagan** [comment]
Hey @bocajspear1 

Could you remove your existing Budibase containers and try running it through the Budi CLI just in case that helps? https://docs.budibase.com/docs/budibase-cli-setup
